src/adagrams.js

```diff
@@ -1,15 +1,202 @@
+const LETTERBANK = [
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "B",
+  "B",
+  "C",
+  "C",
+  "D",
+  "D",
+  "D",
+  "D",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "F",
+  "F",
+  "G",
+  "G",
+  "G",
+  "H",
+  "H",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "J",
+  "K",
+  "L",
+  "L",
+  "L",
+  "L",
+  "M",
+  "M",
+  "N",
+  "N",
+  "N",
+  "N",
+  "N",
+  "N",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "P",
+  "P",
+  "Q",
+  "R",
+  "R",
+  "R",
+  "R",
+  "R",
+  "R",
+  "S",
+  "S",
+  "S",
+  "S",
+  "T",
+  "T",
+  "T",
+  "T",
+  "T",
+  "T",
+  "U",
+  "U",
+  "U",
+  "U",
+  "V",
+  "V",
+  "W",
+  "W",
+  "X",
+  "Y",
+  "Y",
+  "Z",
+];
+
+const LETTERSCORES = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10,
+};
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const randNums = [];
+  // Some concern about this running indefinitely at random...
+  // like if the random number is the same number again and again
+  while (randNums.length < 10) {
+    let num = Math.floor(Math.random() * 98);
+    if (!randNums.includes(num)) {
+      randNums.push(num);
+    }
+  }
+
+  const letters = [];
+  for (let index of randNums) {
+    letters.push(LETTERBANK[index]);
+  }
+  return letters;
 };
 
 export const usesAvailableLetters = (input, lettersInHand) => {
   // Implement this method for wave 2
+  const letters = [...lettersInHand];
```

Comment:
Nice job making a copy of your input before mutating it!  

This is a great practice that can help prevent _so many_ subtle bugs!